package.json

```diff
@@ -25,5 +25,8 @@
     "babel-eslint": "^4.1.7",
     "eslint": "^0.22.1",
     "mocha": "^2.2.5"
+  },
+  "peerDependencies": {
+  	"react": "^0.14.0 || ^15.0.0"
```

Comment:
As far as I know, it should work with 0.13. The HoC doesn't do much more than change the API from the mixin. I'll be updating the version again in my next update.